src/Controller/UserController.php

```diff
@@ -102,9 +103,12 @@ public function delete(Request $request, EntityManagerInterface $entityManager):
             if ($this->isCsrfTokenValid('delete' . $user->getId(), $request->request->get('_token'))) {
                 $entityManager->remove($user);
                 $entityManager->flush();
+                $session = new Session();
```

Comment:
In fact, there's an error getting the session from the Request because at that point of the script, the user data don't exist anymore but the session is alive. So the aim is to re generate a session so as to crush the previous one and to delete it immediately after, because no one is logged in anymore.